# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/udev/hotfix-in6.patch.sparc
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- systemd-257.7/src/basic/linux/ipv6.h.vanilla	2025-07-24 16:29:46.655905315 +0200
+++ systemd-257.7/src/basic/linux/ipv6.h	2025-07-24 16:32:04.972457311 +0200
@@ -5,7 +5,7 @@
 #include <linux/libc-compat.h>
 #include <linux/types.h>
 #include <linux/stddef.h>
-#include <linux/in6.h>
+#include <netinet/in.h>
 #include <asm/byteorder.h>
 
 /* The latest drafts declared increase in minimal mtu up to 1280. */
@@ -18,14 +18,14 @@
  *	*under construction*
  */
 
-#if __UAPI_DEF_IN6_PKTINFO
+#if 0 // __UAPI_DEF_IN6_PKTINFO
 struct in6_pktinfo {
 	struct in6_addr	ipi6_addr;
 	int		ipi6_ifindex;
 };
 #endif
 
-#if __UAPI_DEF_IP6_MTUINFO
+#if 0 //  __UAPI_DEF_IP6_MTUINFO
 struct ip6_mtuinfo {
 	struct sockaddr_in6	ip6m_addr;
 	__u32			ip6m_mtu;
